pkg/controller/usersignupcleanup/predicate.go

```diff
@@ -0,0 +1,26 @@
+package usersignupcleanup
+
+import (
+	toolchainv1alpha1 "github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+	controllerPredicate "sigs.k8s.io/controller-runtime/pkg/predicate"
+)
+
+type UserSignupCleanupPredicate struct {
+	controllerPredicate.Funcs
+}
```

Comment:
You can remove it if you don't want to use it ;-)

Reply:
I've removed the predicate altogether.